### ISSUE TITLE
[Feature Request]: Windows port

### ISSUE THREAD

**Issue by Choochmeque** [sialan-labs/sigram#64]
It would be great to have Windows port of Sigram.

**Comment by realbardia** [comment]
Yes... Currently Sigram using telegram-cli and It can't porting to windows.
But we are porting sigram to the libqtelegram. After that we will porting to windows.
Thank you :)

**Comment by Choochmeque** [comment]
BTW, you can count on me in this ft. req. =)

**Comment by realbardia** [comment]
It is possible to port Sigram to windows and OSX now.
Sigram One is next sigram, based on libqtelegram and completely rewriten.
It will release soon :)

**Comment by Choochmeque** [comment]
When will it release? )

**Comment by realbardia** [comment]
End of December :)
